starport_template/lib/pages/routing_page.dart

```diff
@@ -22,7 +22,6 @@ class _RoutingPageState extends State<RoutingPage> {
   void initState() {
     super.initState();
     initialize();
-    _loadWallets();
```

Comment:
are we sure we want to stop loading wallets on the app start?

Reply:
Yes i discovered this s the cause of multiple account balances being loaded we already call `_loadWallets()` in `initialize()`